views/applications/_form.html

```diff
@@ -0,0 +1,13 @@
+<form action="{{action}}" method="post" enctype="multipart/form-data" class="form" novalidate>
+  <div class="form--field-container">
+    {{form.html | safe}}
+  </div>
+
+  <div class="form--actions">
+    <input type="submit" name="submit" value="{{button}}" class="form--submit">
+    {% if delete %}
+      <input type="submit" name="submit" value="{{config.actions.delete}}" class="form--submit">
+    {% endif %}
+    <a href="../" class="form--cancel">Cancel</a>
+  </div>
+</form>
```

Comment:
I feel that this is likely shared a few places, like the User form? Can we consolidate down to one form?